rules/no-import-test-files.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+const path = require('path');
+const arrify = require('arrify');
+const pkgUp = require('pkg-up');
+const multimatch = require('multimatch');
+const util = require('../util');
+
+const defaultFiles = [
+	'test.js',
+	'test-*.js',
+	'test/**/*.js',
+	'**/__tests__/**/*.js',
+	'**/*.test.js'
+];
```

Comment:
This should be moved to `util` as it's also defined in `no-ignored-test-files` (the one there should be moved too)